Makefile

```diff
@@ -40,6 +40,9 @@ linux: dist/chamber-$(VERSION)-linux-amd64
 dist/chamber-$(VERSION)-linux-amd64: | dist/
 	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -trimpath $(LDFLAGS) -o $@
 
+dist/chamber-$(VERSION)-linux-amd64: | dist/
```

Comment:
shouldn't this be `-linux-arm64` instead of `-linux-amd64`?

Reply:
eagle eyes ! thanks ;)